src/Pim/Bundle/CatalogBundle/Elasticsearch/ProductIndexer.php

```diff
@@ -64,6 +66,33 @@ public function indexAll(array $products, array $options = [])
         }
 
         $this->indexer->bulkIndexes($this->indexType, $normalizedProducts, 'id');
+
+        if (!isset($options['associated_products']) || false === $options['associated_products']) {
```

Comment:
I don't get it, if the `$options['associated_products']` is set to `false`, we still index associated products?